src/panels/config/backup/ha-config-backup-settings.ts

```diff
@@ -243,6 +244,46 @@ class HaConfigBackupSettings extends LitElement {
                   `
                 : nothing}
             </div>
+            ${!this.cloudStatus.logged_in
+              ? html`<ha-card class="cloud-info">
+                  <div class="cloud-header">
+                    <img
+                      .src=${brandsUrl({
+                        domain: "cloud",
+                        type: "icon",
+                        useFallback: true,
+                        darkOptimized: this.hass.themes?.darkMode,
+                      })}
+                      crossorigin="anonymous"
+                      referrerpolicy="no-referrer"
+                      alt="Nabu Casa logo"
+                      slot="start"
+                    />
+                    <span>Home Assistant Cloud backup</span>
```

Comment:
Should it be translated?

Reply:
I am not sure. `Home Assistant Cloud` is a brand. `backup` is a word we could translate. How is it done in french? In german it's just `backup` too as it is common.

Reply:
The translation should be something `{home_assistant_cloud} backup`.
Backup is translated to `sauvergarde` in french.